recipes/simplegmail/setup.patch

```diff
@@ -0,0 +1,21 @@
+From fa75cf48c02f3e56aa9ac1e7d4f99839bb3a6c4f Mon Sep 17 00:00:00 2001
+From: Jan Janssen <jan-janssen@users.noreply.github.com>
+Date: Wed, 26 Jan 2022 08:55:58 -0700
+Subject: [PATCH] Remove bs4
+
+---
+ setup.py | 1 -
+ 1 file changed, 1 deletion(-)
+
+diff --git a/setup.py b/setup.py
+index a191fcf..9b97d44 100644
+--- a/setup.py
++++ b/setup.py
+@@ -12,7 +12,6 @@
+     packages=setuptools.find_packages(),
+     install_requires=[
+         'google-api-python-client>=1.7.3',
+-        'bs4>=0.0.1',
```

Comment:
BTW, we do have a "bs4" package that installs beautifulsoup4. Maybe we don't need this patch.

Reply:
I tried it before without the patch, but then the `pip check` complained.

Reply:
Ah, yeah. pip check is not too smart about that.